final_task/rss_parser/Output_functions.py

```diff
@@ -0,0 +1,227 @@
+import json
+import logging
+import re
+from itertools import groupby
+
+from bs4 import BeautifulSoup
+
+from final_task.rss_parser.Classes.Novelty import Novelty
+
+
+def clean_html(raw_html):
+    """
+    Formatting our String excluding all html-tags making it easier to read our description
+    :param raw_html:
+    :return:
+    """
+    logging.info("Cleaning something from htlm-tags.")
+    cleaner = re.compile('<.*?>')
```

Comment:
What if some text inside feed is wrapped with <> symbols and it's not the html tag? Will it be deleted?

Reply:
It will be deleted, but who will put any text inside of those brackets?
It will be deleted anyway if you use BeautifulSoup, lxml, html or re.

Reply:
Ok, leave it like this then